sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -240,17 +246,24 @@ export interface SubscribeOptions extends OperationOptions, MessageHandlerOption
 
 // @public
 export interface SubscriptionRuleManager {
-    addRule(ruleName: string, filter: boolean | string | CorrelationFilter, sqlRuleActionExpression?: string): Promise<void>;
+    addRule(ruleName: string, filter: boolean | string | CorrelationFilter, sqlRuleActionExpression?: string, options?: OperationOptions): Promise<void>;
```

Comment:
This does result in multiple optional arguments. My preference here would be to have overloads rather than moving the action expression into the options bag. I want to take that call outside the scope of this PR